docs/readmes/proposals/p009_nms_mariadb_migration.md

```diff
@@ -0,0 +1,160 @@
+---
+id: p009_nms_mariadb_migration
+title: NMS MariaDB Migration
+hide_title: true
+---
+
+# Overview
+
+*Status: In Review*\
+*Author: @andreilee*\
+*Last Updated: 02/12*\
+*Targeted Release: 1.5*
+
+This document concerns:
+
+1. Migration of NMS db from MariaDB to to Postgres to share with orc8r
+2. Migration Process
+
+## Goals
+
+Currently the Magma NMS and orc8r are running on separate database
```

Comment:
While what you are mentioning is right here. The ideal state we want to get to is where both NMS and Orc8r use the same DB instance agnostic of whether the underlying DB deployment is mariadb or postgres.